### ISSUE TITLE
Deck Class

### ISSUE THREAD

**Issue by NathanHealea** [cs361-W16/Group21#5]
A deck class needs to be created
## Requirements:
1. Must hold an array of 52 cards
2. Shuffle method
3. Reset method
4. Create deck method
5. Hold card Count.
6. Deal method. Only deals one card.

**Comment by foulgerd** [comment]
Right, but then we also need a card class. Well, one that can generate suits and a value.

**Comment by NathanHealea** [comment]
Yes I have created that and closed it.

**Comment by NathanHealea** [comment]
This has been updated so that Deck does not hold the count. It has two function that determine the count of cards in the deck.
